styles/globals.css

```diff
@@ -57,4 +57,17 @@ h2 {
 
 .button-shadow {
   box-shadow: 3px 3px 6px 0px #0000003d;
-}
\ No newline at end of file
+}
+/** Stories */
```

Comment:
Why convert this into CSS? If it seemed like a lot of complicated logic inside `className` you can always extract individual state styles into separate tailwind string constants or build the `className` in a standalone function. 

But you still have the styles co-located and scoped to the component and are forced to stick with the Tailwind design system limitations (which is a good thing).